helpers/aws/functions.js

```diff
@@ -227,6 +227,17 @@ function crossAccountPrincipal(principal, accountId) {
     return false;
 }
 
+
+function hasFederatedUserRole(policyDocument) {
+    // true iff every statement refers to federated user access 
+    for (let statement of policyDocument) {
+        if (!statement.Action.includes('sts:AssumeRoleWithSAML') && !statement.Action.includes('sts:AssumeRoleWithWebIdentity')){
```

Comment:
should not we check here statement.Action exists first?

Reply:
Yes we should